fix(#155): fix for broken nomenclature after moving data to a folder collection

rapzo opened the pull request:
- added integrated terminal as option for debug

rapzo commented:
test this please

rapzo commented:
Damn it, the comment trick doesn't work.
Dunno why it failed. I have it deployed here: https://vibrant-sammet-885f51.netlify.com/
Everything seems ok, even the admin:
![Screenshot 2020-03-19 at 14 47 18](https://user-images.githubusercontent.com/147788/77079925-9ae52f00-69f0-11ea-937d-c15525beac4b.png)

philsturgeon commented:
Ok I removed all mention of charity_url (that concept has not gone anywhere yet (#150) and the build is working, _but_ ordering was how we worked out "recent links".

<img width="922" alt="image" src="https://user-images.githubusercontent.com/67381/77084670-324d8080-69f7-11ea-9679-024c6a4fd695.png">

Any ideas how to achieve this one in netlify cms?

philsturgeon commented:
Ah looks like no dice https://github.com/netlify/netlify-cms/issues/54

rapzo commented:
Ok, had some typos and stuff there, all green.

Ah! I knew order was important! So, maybe we should look at the slug thing then. I'll rerun the script to parse the content with a different name and will look on how to include an identifier that isn't a date.

rapzo commented:
Well, the [slug](https://www.netlifycms.org/docs/configuration-options/#slug) covers the ordering of new posts/links, but not the old ones... Maybe i'll browse through commit history and come up with something.

rapzo commented:
That was not my smartest ideia ahahaha. I guess the 1st ones never are.
Gonna fake them dates while respecting the order in the original `links.yaml` file. I guess it will sort it out as expected.

rapzo commented:
So, I added the slug configuration and mocked a prefix for all links added to keep the initial order.
Also, I updated the script to create them, https://gist.github.com/rapzo/7437aafed4562bc1386068d7e2f98f36
Just in case you want to fiddle with its form.

rapzo commented:
I believe this was the original order, am i right?
![Screenshot 2020-03-20 at 11 31 51](https://user-images.githubusercontent.com/147788/77160020-67f97480-6a9e-11ea-82e9-518ae98f2f33.png)

rapzo commented:
Was adding links possible with a single file? I got lots of issues as was,
while using the netlify-cms. Were y'all editing the file and doing a
release?
Maybe i missed something... I was just trying to ease the process.
The slug is automatic, by the way. No need to worry about it unless it is
done via editing files and not using the CMS.

On Fri, 20 Mar 2020 at 14:37, Phil Sturgeon <notifications@github.com>
wrote:

> Merged #157 <https://github.com/philsturgeon/awesome-earth/pull/157> into
> master.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/philsturgeon/awesome-earth/pull/157#event-3149584473>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABECTHDEJDRH4WU63CRQDTRIN5ULANCNFSM4LPKLPVQ>
> .
>


-- 
Rui Pedro Lima
